### PR TITLE
Add archive-one-person feature

### LINKED ISSUE #32: As a store manager, I can archive some empolyees' contact details
...so that I still can retrieve them if needed in the future

### PR DESCRIPTION
**Pull request by yanbingtao**
Allows user to use "archive [INDEX]" command to archive a person

### CODE REVIEW

**Review by Persdre — APPROVED** [review]
LGTM

File: src/main/java/seedu/address/logic/commands/ArchiveCommand.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+    private final Index targetIndex;
+
+    /**
+     * Constructs an ArchiveCommand.
+     *
+     * @param targetIndex the index number shown in the displayed person list.
+     */
+    public ArchiveCommand(Index targetIndex) {
+        requireAllNonNull(targetIndex);
```
Comment (ureshiiYing) [inline src/main/java/seedu/address/logic/commands/ArchiveCommand.java:36]: Since it's just one argument, `requireNonNull(targetIndex)` will do.

**Review by ureshiiYing — APPROVED** [review]
Overall, LGTM. The part of the code commented can be edited in the next iteration, since it is a small change. You should also add tests for ArchiveCommandParser.

File: src/main/java/seedu/address/logic/commands/ArchiveCommand.java (hunk 1 above)
Comment (yanbingtao) [reply src/main/java/seedu/address/logic/commands/ArchiveCommand.java:36]: Thanks for pointing out! Will edit it and change it in the next iteration.

**Pull request merged**